modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/PassThroughConstants.java

```diff
@@ -234,4 +236,15 @@
 
     //Check for invalid json message by parsing the input message
     public static final String FORCE_JSON_MESSAGE_VALIDATION = "force.json.message.validation";
+
+    /**
+     * constants for the EI observability
+     */
+    public static final String CORRELATION_LOG_STATE_PROPERTY = "correlationLogState";
```

Comment:
Shall we add a description for the static constants?

Reply:
fixed in https://github.com/wso2/wso2-synapse/pull/1305